eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -76,12 +76,19 @@ jobs:
         Linux:
           OSName: "Linux"
           OSVmImage: "ubuntu-16.04"
-        Windows:
+          TFM: netcoreapp2.1
+        Windows_NetCoreApp:
           OSName: "Windows"
           OSVmImage: "vs2017-win2016"
+          TFM: netcoreapp2.1
+        Windows_NetFramework:
+          OSName: "Windows"
+          OSVmImage: "vs2017-win2016"
+          TFM: net461
         MacOs:
           OSName: "MacOS"
           OSVmImage: "macOS-10.13"
+          TFM: netcoreapp2.1
```

Comment:
Can you please rename TFM to TargetFramework to make it more explicit and also alight with what it is called in the projects?